body-parser/body-parser-tests.ts

```diff
@@ -0,0 +1,22 @@
+import * as express from 'express';
+import {
+    json,
+    raw,
+    text,
+    urlencoded,
+} from 'body-parser';
+
+const app = express();
+
+express.use(json());
+express.use(raw());
```

Comment:
You're initializing the variable ```app``` but not actually using it. ```express``` is the constructor, not the instance, which is why you're getting errors.